department-of-veterans-affairs/content-build
# VACMS-14541: Add Reusable QAs component for R&S detail pages

## Description
We want to create a reusable QA component that can be utilized across different benefits details pages.

relates to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14541

## Testing done & Screenshots

Testing with different scenarios of QA group implementation.
Tugboat instance to verify:  https://web-puvomwkxsno6aa4slenuvyqhhoin1hox.ci.cms.va.gov/resources/test-test-1/

Placing QA Group in an ordered list:
![order1](https://github.com/department-of-veterans-affairs/content-build/assets/42885441/e04d26dc-7a03-46db-ac72-2d5fe958df9b)

QA Group alone:
![qa_alone](https://github.com/department-of-veterans-affairs/content-build/assets/42885441/e524abaa-012b-4524-94f7-5f5c67dd0012)


QA Group at the end of the list:
![Test_QA_Group_At_The_End___Veterans_Affairs](https://github.com/department-of-veterans-affairs/content-build/assets/42885441/b5239665-b48a-40fc-b22e-46f40d799093)




## QA steps

What needs to be checked to prove this works?  
Verify that the QA group is valid in ordered list
What needs to be checked to prove it didn't break any related things?  
Check against R&S pages that have QA Groups embedded in details pages.

## Definition of done
